src/server/helpers/memcached.js

```diff
@@ -0,0 +1,30 @@
+import Memcached from 'memcached';
+
+import { conf } from '../helpers/config';
+
+let memcached = null;
+
+const getMemcachedStub = () => {
+  return {
+    get: (key, callback) => callback(),
```

Comment:
Just a simple 'pass-through' stub - nothing is cached, only callback is called (with undefined value).